services-camel/src/main/java/edu/unc/lib/dl/services/camel/cdrEvents/CdrEventRouter.java

```diff
@@ -38,6 +49,12 @@
     @BeanInject(value = "cdrEventToSolrUpdateProcessor")
     private CdrEventToSolrUpdateProcessor cdrEventToSolrUpdateProcessor;
 
+    private static final List<String> solrAllowedActions = asList(MOVE.toString(), REMOVE.toString(), ADD.toString(),
```

Comment:
again, these could be alphabetized to improve readability and maintainability

Reply:
a->z'ed